rakelib/artifacts.rake

```diff
@@ -201,7 +201,7 @@ namespace "artifact" do
     # TODO(sissel): Invoke Pleaserun to generate the init scripts/whatever
 
     out.name = "logstash"
-    out.version = LOGSTASH_VERSION
+    out.version = LOGSTASH_VERSION.gsub(/\.([[:alpha:]])/, '~\1')
```

Comment:
This affects _all_ packages built by fpm in rake. I'll setup a centos box to test this also.